jenkins/Jenkinsfile

```diff
@@ -0,0 +1,109 @@
+def secrets = [
+  [path: 'secret/jenkins/github', engineVersion: 2, secretValues: [
+    [envVar: 'PRIVATE_REPO_GITHUB_TOKEN', vaultKey: 'private_token']]],
```

Comment:
In case of `cloudify-ui-common` we actually don't need to have token with private repositories access as `cloudify-ui-common` is public.

Reply:
ok, adding a public token to vault also